unit-tests/dds/test-librs-formats-converison.py

```diff
@@ -0,0 +1,178 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+
+#test:donotrun:!dds
+
+from rspy import log, test
+import pyrealsense2 as rs
+import dds
+
+rs.log_to_console( rs.log_severity.debug )
+log.nested = 'C  '
+
+context = rs.context( '{"dds-domain":123,"dds-participant-name":"test-formats-conversion"}' )
+only_sw_devices = int(rs.product_line.sw_only) | int(rs.product_line.any_intel)
+
+import os.path
+cwd = os.path.dirname(os.path.realpath(__file__))
+remote_script = os.path.join( cwd, 'formats-conversion-server.py' )
+with test.remote( remote_script, nested_indent="  S" ) as remote:
+    remote.wait_until_ready()
+
+    # Tests setup
+    remote.run( 'create_server()' )
+    n_devs = 0
+    for dev in dds.wait_for_devices( context, only_sw_devices ):
+        n_devs += 1
+    test.check_equal( n_devs, 1 )
+    sensors = {sensor.get_info( rs.camera_info.name ) : sensor for sensor in dev.query_sensors()}
+    #
+    #############################################################################################
+    #
+    with test.closure( "Test Y8 conversion", on_fail=test.ABORT ):
+        if test.check( 'Y8-sensor' in sensors ):
+            sensor = sensors.get('Y8-sensor')
+            profiles = sensor.get_stream_profiles()
+
+            test.check_equal( len( profiles ), 1 ) # Y8 stays Y8, for infrared stream indexed 1
+            test.check_equal( profiles[0].format(), rs.format.y8 )
+            test.check_equal( profiles[0].stream_index(), 1 )
+    #
+    #############################################################################################
+    #
+    with test.closure( "Test Y8I conversion", on_fail=test.ABORT ):
```

Comment:
Why abort? You want to stop the test on the first failure?

Reply:
Changed to RAISE

Reply:
Why RAISE? It'll raise another exception, which will do pretty much the same as ABORT... Why not just leave the default behavior?

Reply:
Now I understand. Removed and left to default behaviour.